### PR TITLE
Add vital information for getting this to work

### PR DESCRIPTION
**Pull request by nacholibre**

### CODE REVIEW

File: Resources/doc/overriding_controllers.rst [hunk 1]
```diff
@@ -87,6 +87,28 @@ the base controller and adds logging a new user registration to it.
...
+    use Symfony\Component\Config\Loader\LoaderInterface;
+
+    class AppKernel extends Kernel
+    {
+        public function registerBundles()
+        {
+            $bundles = array(
+                //..
+                new FOS\UserBundle\FOSUserBundle(),
+                new Acme\UserBundle\AcmeUserBundle(),
```
Comment (stof) [inline Resources/doc/overriding_controllers.rst]: this should be placed near the place asking to create the bundle

File: Resources/doc/overriding_controllers.rst [hunk 2]
```diff
@@ -30,6 +30,25 @@ bundle named ``AcmeUserBundle`` that declares itself a child of FOSUserBundle.
 
     The Symfony Framework only allows a bundle to have one child. You cannot
     create another bundle that is also a child of FOSUserBundle.
+    
+Then you need to add AcmeUserBundle in AppKernel.php.
```
Comment (Ener-Getick) [inline Resources/doc/overriding_controllers.rst]: What about `Then enable the bundle in the kernel:` ?

Comment (nacholibre) [inline Resources/doc/overriding_controllers.rst]: Even better.

**Comment by XWB** [comment]
Please open this PR against the master branch.

**Pull request closed without merging**